pkg/ingester/ingester_test.go

```diff
@@ -1217,10 +1217,10 @@ func TestIngester_Push(t *testing.T) {
 
 				// We expect no error on any request except the last one
 				// which may error (and in that case we assert on it)
-				if idx < len(testData.reqs)-1 {
+				if idx < len(testData.reqs)-1 || testData.expectedErr == nil {
 					assert.NoError(t, err)
 				} else {
-					assert.Equal(t, testData.expectedErr, err)
+					assert.ErrorContains(t, err, testData.expectedErr.Error())
```

Comment:
Are we ok by just checking the error message here? Is there any caller relying on the type and unwrapping of it? Can we unwrap by using `errors.As` and check the equality of wrapped message?

Reply:
This modification is actually not needed. It is a leftover from a previous version of the PR.